api/views.py

```diff
@@ -62,19 +62,28 @@ def tasks_list_all(request: HttpRequest):
 
     count = None
     start = None
+    order = ["-created_at", "-id"]
```

Comment:
`order = ("-created_at", "-id")`, otherwise it looks like there is a reason why it is mutable, e.g. because you try to append to it somewhere :)